### DISCUSSION TITLE
关于 ant-design-vue是否支持 vue2？

### DISCUSSION THREAD

**Discussion by yujiahan2018** [vueComponent/ant-design-vue#6122] [Q&A]
当vue环境是v2，在使用npm命令 npm install ant-design-vue --save 安装 ant-design时，v1~v3均报下图错误，疑似版本太高，而使用vue3，安装成功。所以想核实一下目前ant-design-vue是否还支持vue2版本
![image](https://user-images.githubusercontent.com/40731243/204435607-18ace27f-ce61-4362-9ffa-73e56ffb3a9f.png)

附环境中vue版本：
![image](https://user-images.githubusercontent.com/40731243/204436134-306d7cae-1c40-408f-ba10-e0f243c8838c.png)

**Comment by TLovers** [comment]
![image](https://user-images.githubusercontent.com/26561694/207049233-2c933693-72fc-4ee9-b9ae-d2f0aa0b5395.png)
![image](https://user-images.githubusercontent.com/26561694/207049735-fc800626-f76d-4ae3-b8f4-63b5b01ddaf9.png)
可以指定安装1.x 分支的
